core/domain/user_jobs_one_off.py

```diff
@@ -491,6 +491,45 @@ def reduce(key, values):
         yield (key, len(values))
 
 
+class RemoveFeedbackThreadIDsOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """Job that deletes the feedback_thread_ids from the UserSubscriptionsModel.
```

Comment:
Please add a line saing that this is just one-off job and can be removed after January release.

Reply:
Done.